### ISSUE TITLE
Early Testers Airdrop

### ISSUE THREAD

**Issue by crisgarner** [cryptexfinance/contracts#79]
- [x] Create contracts
- [x] Make tests
- [x] Generate airdrop file

**Comment by crisgarner** [comment]
Deployment of this will depend on #34

**Comment by tommatzgithub** [comment]
Can also see if we can get it included on here https://twitter.com/defi_airdrops?s=11

**Comment by voith** [comment]
No longer needed.
